app/src/main/java/com/rentminder/Icon&Text.kt

```diff
@@ -1,104 +1,87 @@
 package com.rentminder
 
-import androidx.compose.foundation.layout.Arrangement
 import androidx.compose.foundation.layout.Row
-import androidx.compose.foundation.layout.padding
 import androidx.compose.foundation.layout.size
 import androidx.compose.material.Icon
 import androidx.compose.material.Text
-import androidx.compose.material.icons.Icons
-import androidx.compose.material.icons.outlined.Home
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
-import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
 import androidx.compose.ui.res.painterResource
-import androidx.compose.ui.text.TextStyle
-import androidx.compose.ui.text.style.TextDecoration
 
 @Composable
 fun RentIconText() {
     Row(verticalAlignment = Alignment.CenterVertically){
-        Icon(painterResource(
-            id = R.drawable.outline_house_24
-        ),
+        Icon(
+            painterResource(id = R.drawable.outline_house_24),
             contentDescription = "Rent Icon",
-            modifier = Modifier
-                .size(30.dp)
+            modifier = Modifier.size(30.dp)
         )
         Text(
             text = "Rent",
-            fontSize = (26.sp)
+            fontSize = 26.sp,
         )
     }
 }
 
 @Composable
 fun ElectricIconText() {
     Row(verticalAlignment = Alignment.CenterVertically){
-        Icon(painterResource(
-            id = R.drawable.outline_lightbulb_24
-        ),
+        Icon(
+            painterResource(id = R.drawable.outline_lightbulb_24),
             contentDescription = "Light Bulb Icon",
-            modifier = Modifier
-                .size(30.dp)
+            modifier = Modifier.size(30.dp)
         )
         Text(
             text = "Electric/Gas",
-            fontSize = (26.sp)
+            fontSize = 26.sp,
```

Comment:
Added trailing comments to improve consistency across the project, as some parameters had them and some did not. I added them to all parameters to also make it easier to add more later on. This increases the speed of adding/removing parameters because you never have to second guess whether you should put a comma or not.